tests/integrated/test-coordinates-initialization2/test-coordinates-initialization.cxx

```diff
@@ -0,0 +1,31 @@
+/*
+ * Test of initialization of Coordinates objects in Mesh::coords_map
+ */
+
+#include "bout.hxx"
+#include "optionsreader.hxx"
+
+int main(int argc, char** argv) {
+
+  // Initialize BOUT++
+  BoutInitialise(argc,argv);
```

Comment:
It's not clear what extra this test brings over the other -- presumably if the other doesn't fail this one shouldn't (beyond being slower)?

Reply:
> if the other doesn't fail this one shouldn't

That's true. Sorry, I probably shouldn't have put both tests in.
The reason for having this test is to show that if we require `BoutInitialise()` to be called, then 
the `getCoordinates()` call in `BoutMesh::load()` (added in ed81860) is not 'necessary' (see discussion on #1369). If we're happy with ed81860, then `test-coordinates-initialization2` is indeed redundant.